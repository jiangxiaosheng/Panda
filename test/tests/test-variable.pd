var x = 1

func main() {
	var s1 = "hello"
	print(s1)
	var s2 = "world"
	print(s1 + s2)
}