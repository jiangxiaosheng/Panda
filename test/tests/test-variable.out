Sementically checked program: 

var x: int(int : 1)

void main() {
}

